src/set-from-request.js

```diff
@@ -0,0 +1,8 @@
+import { set } from './utils'
+
+export default function (requestKey, path) {
+  return {
+    [`${requestKey}_SUCCESS`]: (state, action) => set(path, action.payload, state),
+    [`${requestKey}_FAILURE`]: (state) => set(path, null, state)
```

Comment:
@davemcorwin how is the user going to determine between an error and a success response in the state?